on_device_training/mobilevit/README.md

```diff
@@ -0,0 +1,47 @@
+# MobileVIT End-to-End Example for On-Device Training
+This folder contains an end-to-end example for on-device training in a C# console app using the ONNXRuntime Training libraries.
+
+On-device training contains two steps: an "offline" pre-processing step that happens off of the device in question, and the training step that happens on the device. 
+
+## About the task
+The task for this example is facial expression recognition, categorizing an image into one of the following emotions:
+1. neutral
+2. happy
+3. sad
+4. surprise
+5. fear
+6. disgust
+7. angry
+
+The huggingface MobileVIT xx-small model is fine-tuned on the huggingface DiffusionFER dataset to perform this classification.
+
+## Contents
+The `mobilevit_offline.ipynb` Python notebook creates the necessary training artifacts for training. The following generated files & folders should be **migrated or moved** from the "offline" device to the device to be used for training:
+1. The training ONNX model (default name will be `training_model.onnx`)
+2. The eval ONNX model (default name will be `eval_model.onnx`)
+3. The optimizer ONNX model (default name will be `optimizer_model.onnx`)
+4. The checkpoint files (default name of the folder will be `checkpoint`, and all files in this folder should be moved to the device to be used for on-device training)
+
+The `mobilevit_console` folder contains the C# console app that performs the training & inferencing for this task.
+
+Once the training step is finished, one more ONNX model file will be generated. Its default name will be `mobilevit_console/mobilevit_console/bin/Debug/net6.0/trained_mobilevit.onnx`.
+
+## Prerequisites for offline processing step
+Make sure PyTorch, transformers, and numpy are installed into your Python environment.
+
+Check the [instructions for installing the onnxruntime-training Python package](https://onnxruntime.ai/) & install it.
+
+## Prerequisites for on-device step
```

Comment:
Break this down into 2 Preprocessing the data and Running the example. Make it clear that ideally user data is used for training but for the sake of this example we will be using the DiffusionFER dataset.